### PR TITLE
English names for transport network descriptions

### LINKED ISSUE #286: Language of name of places
Are name of cities and regions written in English or in the local language?
Some places are written in English (Upper Austria, Saxony) while other are written in local language (Corse, Roma, Milano, Venezia).
Let's decide a standard and adapt all names.

### PR DESCRIPTION
**Pull request by Airon90**
Fixed #286

### CODE REVIEW

File: res/values/strings.xml [hunk 1]
```diff
@@ -276,23 +276,23 @@ there are any).</string>
...
     <string name="np_desc_vbn" tools:keep="@string/np_desc_vbn">Lower Saxony, Bremen, Bremerhaven, Oldenburg (Oldenburg), Osnabrück</string>
     <string name="np_desc_vgn">Nuremberg, Fürth, Erlangen, Bayreuth, Bamberg, Schwabach</string>
     <string name="np_desc_vgn_networks" translatable="false">VAG, DB, ESTW, infra fürth, Stadtwerke Bayreuth, Stadtwerke Schwabach, STWB, GPV, GkV</string>
     <string name="np_desc_vgs">Saarland, Saarbrücken</string>
     <string name="np_desc_vms">Mittelsachsen, Chemnitz</string>
     <string name="np_desc_vmv">Mecklenburg-Vorpommern, Schwerin</string>
-    <string name="np_desc_vrn">Baden-Württemberg, Rheinland-Pfalz, Mannheim, Mainz, Trier</string>
-    <string name="np_desc_vrr">North Rhine-Westphalia, Köln, Bonn, Essen, Dortmund, Düsseldorf, Münster, Paderborn, Höxter</string>
+    <string name="np_desc_vrn">Baden-Württemberg, Rhineland-Palatinate, Mannheim, Mainz, Trier</string>
+    <string name="np_desc_vrr">North Rhine-Westphalia, Cologne, Bonn, Essen, Dortmund, Düsseldorf, Münster, Paderborn, Höxter</string>
```
Comment (grote) [inline res/values/strings.xml]: Düsseldorf, Münster and Höxter are like this in English?

File: res/values/strings.xml [hunk 2]
```diff
@@ -276,23 +276,23 @@ there are any).</string>
...
     <string name="np_desc_vrs">Cologne, Bonn</string>
     <string name="np_desc_vvm">Swabia, Mittelschwaben, Krumbach, Günzburg, Memmingen</string>
```
Comment (grote) [inline res/values/strings.xml]: Swabia and Mittelschwaben doesn't look right either. It is mixed German and English.

**Comment by grote** [comment]
Thanks for this PR as well! I made two comments, but there's probably more I didn't check. I'd prefer if this PR finalizes the English names, so we don't need many more after it.

File: res/values/strings.xml (hunk 2 above)
Comment (Airon90) [inline res/values/strings.xml]: [It seems](https://en.wikipedia.org/wiki/Mittelschwaben) that "Mittelschwaben" isn't been anglicized. Moreover, [even in Danish](https://da.wikipedia.org/wiki/Mittelschwaben) it isn't been translated

File: res/values/strings.xml (hunk 1 above)
Comment (Airon90) [inline res/values/strings.xml]: [Düsseldorf](https://en.wikipedia.org/wiki/Düsseldorf), [Münster](https://en.wikipedia.org/wiki/Münster) and [Höxter](https://en.wikipedia.org/wiki/Höxter) are used in English ;)

**Comment by Airon90** [comment]
FYI in this PR I changed only French and German places as other places have been translated in the other PR ;)

**Comment by grote** [comment]
Thanks for checking all those names!

**Comment by Airon90** [comment]
I'm not a (good) coder so I give a help as far as I can ;)
Thank you for your work!

**Pull request merged**